yktrace/src/tir.rs

```diff
@@ -330,12 +340,22 @@ impl VarRenamer {
         self.offset
     }
 
+    fn init_acc(&mut self, num_locals: usize) {
+        if self.acc.is_none() {
+            self.acc.replace(num_locals as u32);
+        }
+    }
+
     fn enter(&mut self, num_locals: usize, dest: Place) {
-        // When entering an inlined function call update the current offset by adding the number of
-        // assigned variables in the outer context. Also add this offset to the stack, so we can
-        // restore it once we leave the inlined function call again.
-        self.offset += num_locals as u32;
+        // When entering an inlined function call set the offset to the current accumulator. Then
+        // increase the accumulator with the number of locals in the current function. Also add the
```

Comment:
Minor wording tweak:
"increase the accumulator with the number of locals" -> "increment the accumulator by the number of locals"

Reply:
Fixed in 61cadb7